application/event_processor/event_processor.py

```diff
@@ -156,6 +153,7 @@ def lambda_handler(event: SQSEvent, context: LambdaContext) -> None:
         return
 
     event_processor.get_change_requests()
+    disconnect_dos_db()
```

Comment:
this need to be in a finally block so that it is still attempeted in event of intermediate code throwing an exception